src/components/experiment-tracking/run-details-modal/run-details-modal.scss

```diff
@@ -18,3 +18,15 @@
   text-align: right;
   width: 100%;
 }
+
+.version-reminder-and-run-details-button-wrapper {
```

Comment:
what is this change for?

Reply:
This is for the version reminder and buttons to align on the same level:

<img width="903" alt="image" src="https://github.com/kedro-org/kedro-viz/assets/90610031/663872a7-49d9-4763-8133-1b42bf913c2e">

I see the `.run-details-modal-button-wrapper` class was also being used by `run-details-modal.js` so I created a new class for `settings-modal.js` to avoid any side-effects.

Reply:
Let me know if you have any suggestions I can change it here quickly as well.